Formula/protobuf.rb

```diff
@@ -4,6 +4,7 @@ class Protobuf < Formula
   url "https://github.com/protocolbuffers/protobuf/releases/download/v3.17.3/protobuf-all-3.17.3.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/protocolbuffers/protobuf/releases/download/v3.19.1/protobuf-all-3.19.1.tar.gz"
```

Reply:
#85282

Reply:
Please retry now that #87287 has landed.

Reply:
Is #87287 not this PR?